partition/roles/ipmi-catcher/tasks/main.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- name: deploy ipmi-catcher service
+  include_role:
+    name: ansible-common/roles/systemd-docker-service
+  vars:
+    systemd_start: "{{ ensure_started.ipmi_catcher|default(true) }}"
+    systemd_service_name: ipmi-catcher
+    systemd_docker_image_name: "{{ ipmi_catcher_image_name }}"
+    systemd_docker_image_tag: "{{ ipmi_catcher_image_tag }}"
+    systemd_docker_network: host
+    systemd_docker_volumes:
+      - /var/lib/dhcp:/var/lib/dhcp:ro
+    systemd_docker_cap_add:
+      - sys_admin
+    systemd_service_after: docker.service
+    systemd_service_requires: docker.service
+    systemd_service_environment:
+      TZ: "{{ timezone }}"
+      IPMI_CATCHER_LEASE_FILE: /var/lib/dhcp/dhcpd.leases
+      IPMI_CATCHER_PARTITION_ID: "{{ partition_id }}"
+      IPMI_CATCHER_METAL_API_URL: "{{ api_protocol }}://{{ api_addr }}:{{ api_port }}{{ api_basepath }}"
```

Comment:
Also these variables should get prefixed. Also a candidate for partition-defaults, I think